index.js

```diff
@@ -71,11 +71,17 @@ function iconfontCSS(config) {
 			});
 		}
 
-		currentCodePoint = currentGlyph.toString(16).toUpperCase();
+		var fileName = path.basename(file.path, '.svg');
+		
+		if (config.fixedCodepoints && config.fixedCodepoints[fileName]){
+			currentCodePoint = config.fixedCodepoints[fileName].toString(16).toUpperCase();
+		} else {
+			currentCodePoint = (currentGlyph++).toString(16).toUpperCase();
```

Comment:
Nope, I've removed counter increase happening several lines later. So currentGlyph gets increased only if there's no fixed codepoint defined for current file.